src/wallet/wallet_rpc_server_error_codes.h

```diff
@@ -78,3 +78,4 @@
 #define WALLET_RPC_ERROR_CODE_ATTRIBUTE_NOT_FOUND    -45
 #define WALLET_RPC_ERROR_CODE_ZERO_AMOUNT            -46
 #define WALLET_RPC_ERROR_CODE_INVALID_SIGNATURE_TYPE -47
+#define WALLET_RPC_ERROR_CODE_DISABLED               -48
```

Comment:
```suggestion
#define WALLET_RPC_ERROR_CODE_MULTISIG_DISABLED      -48
```

Reply:
For `WALLET_RPC_ERROR_CODE_DISABLED` are you trying to add a generic 'disabled' code, or should it be `WALLET_RPC_ERROR_CODE_MULTISIG_DISABLED`?